spec/liquid_tags/unified_embed_spec.rb

```diff
@@ -3,6 +3,9 @@
 RSpec.describe UnifiedEmbed do
   subject(:unified_embed) { described_class }
 
+  let(:user) { create(:user) }
+  let(:article) { create(:article, user_id: user.id) }
```

Comment:
Are these being used? It looks like they might've been added to get an article but then, during editing/refactoring, the reason for them existing may have been removed.

Reply:
Hey @jgaskins I use the `article` in the following spec on lines `42 to 45`:
```ruby
it "returns Forem Link for a forem url" do
   expect(described_class.find_liquid_tag_for(link: URL.url + article.path))
     .to eq(LinkTag)
end
```
Now that I look again, I don't need `user` so I will remove 👍🏾

Reply:
Ooooh, I was looking for it and looked right past it. I guess it's time for another coffee. :joy: